src/main/resources/application-generateDDL.yaml

```diff
@@ -0,0 +1,4 @@
+spring.jpa.properties.javax.persistence.schema-generation.create-source: metadata
+spring.jpa.properties.javax.persistence.schema-generation.scripts.action: create
+spring.jpa.properties.javax.persistence.schema-generation.scripts.create-target: src/main/resources/db/migration/V1_1__create_from_hibernate.sql
```

Comment:
Is this yaml file just a temporary thing to get the DDL from our entities using hibernate?

Reply:
I think you will need to add the flyway script to add the corresponding audit table for the Notification entity that is `@Audited`